packages/base/src/index.js

```diff
@@ -392,7 +386,13 @@ module.exports = {
     'jsdoc/require-returns-type': 'error',
     'jsdoc/require-yields': 'error',
     'jsdoc/require-yields-check': 'error',
-    'jsdoc/tag-lines': 'error',
+    'jsdoc/tag-lines': [
+      'error',
+      'any',
+      {
+        startLines: 1,
+      },
+    ],
```

Comment:
AFAIK this matches the previous behaviour, where one line between the description and the first tag is required.